src/tools/compiletest/src/header/tests.rs

```diff
@@ -430,6 +430,8 @@ fn wasm_special() {
     for (target, pattern, ignore) in ignores {
         let mut config = config();
         config.target = target.to_string();
+        dbg!(target, pattern, ignore);
+        dbg!(config.target_cfgs());
```

Comment:
You may want to remove these since they generate a large amount of output.

Reply:
Whelp that wasn't supposed to be left there, I'll remove it tomorrow.